pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -695,6 +696,17 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 			}
 
 			// Set replicas
+			// if the configuration use only one replica we increment
+			if serverSet.JbpmCluster != nil && *serverSet.JbpmCluster {
```

Comment:
this seems redundant... would the following be sufficient instead?
```golang
if *serverSet.JbpmCluster {
```

Reply:
ok

Reply:
Ok